examples/interpretation/notebooks/utilities.py

```diff
@@ -328,23 +328,23 @@ def download_pretrained_model(config):
         raise NameError(
             "Unknown dataset name. Only dutch f3 and penobscot are currently supported."
         )
-
+    
     if "hrnet" in config.MODEL.NAME:
         model = "hrnet"
     elif "deconvnet" in config.MODEL.NAME:
         model = "deconvnet"
-    elif "unet" in config.MODEL.NAME:
-        model = "unet"
+    elif "resnet" in config.MODEL.NAME:
```

Comment:
shouldn't this be "seresnet" or "resnet_unet" ?
we might have other models that have a resent backbone

Reply:
I disabled unet as we don't have / provide a pre-trained model with it